src/plugin/tasks/serverless/snippets/serverless/test/0-sls-cf-stack.spec.js

```diff
@@ -5,6 +5,7 @@ const path = require('path');
 const yaml = require('yaml-boost');
 const fs = require('smart-fs');
 const expect = require('chai').expect;
+const Joi = require('joi-strict');
```

Comment:
Not sure how I feel about adding the join dependency for this...? Can we just do number.isInteger instead?